### ISSUE TITLE
Some features to analyse 2D wall loads

### ISSUE THREAD

**Issue by Otsooni** [ascot4fusion/ascot5#110]
Currently, Ascot can only analyse wall loads for 3D walls. However, 2D walls are widely used and there exists meaning in analysing the poloidal distribution of the wall loads. These methods also make it easier to compare to LOCUST results (since they at least used to have only 2D wall load analysis support).

If you have any suggestions on the implemented methods, put them here.

Currently done:
-Get the areas of each revolving line strip segment
-get the length of each line strip

**Comment by Otsooni** [comment]
Currently implemented so that the runmixin methods of calculating heat loads work for also 2D wall. However, currently the incident angle will not work(will not return anything meaningful), since for 2D wall the incident angle is determined in only R and z (and phi would be determined on the collision point of the marker, but that is not currently used for 3D detection).

Therefore, iangle for 2D walls will not currently result in anything meaningful. Should the code say something of this to the user, or what is your opinion @miekkasarki ?

**Comment by Otsooni** [comment]
What I mean by that is that I did not modify the runmixin method of calculating heat loads. The runmixin only needed to call `wall.area()`. I implemented that area method for 2D walls. However, the heatload method also wants surface normals of the walls (which are undefined for the 2D walls due to phi), so currently the iangle does not work with the method. Moreover, I am not certain of the usefulness of that metric for 2D walls. (If you are certain it is useful, I can modify the runmixin heatload method to get the correct normals for 2D walls. The reason I originally did not do it is because the method is getting too large/convoluted.)

My question to you was, (assuming no modification to the heatload code) should I give a note that the 2D incident angle is not defined.
